tests/countries/test_kiribati.py

```diff
@@ -0,0 +1,199 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.kiribati import Kiribati, KI, KIR
+from tests.common import CommonCountryTests
+
+
+class TestKiribati(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(1980, 2050)
+        super().setUpClass(Kiribati, years=years)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(Kiribati(years=1979))
+    def test_country_aliases(self):
+        self.assertAliases(Kiribati, KI, KIR)
+
+    def test_new_years(self):
+        name = "New Year's Day"
+        self.assertHolidayName(name, (f"{year}-01-01" for year in range(1980, 2050)))
+        dt = (
+            "2017-01-02",
+            "2023-01-02",
+        )
+        self.assertHolidayName(f"{name} (observed)", dt)
+        self.assertNoNonObservedHoliday(dt)
```

Comment:
```suggestion
        obs_dt = (
            "2011-01-03",
            "2012-01-02",
            "2017-01-02",
            "2022-01-03",
            "2023-01-02",
        )
        self.assertHolidayName(f"{name} (observed)", obs_dt)
        self.assertNoNonObservedHoliday(obs_dt)
```
And same for rest similar cases (include observed dates from Sat; variable name).

Reply:
Yes, I completely agree!